apis/v1alpha1/gateway_types.go

```diff
@@ -85,8 +85,8 @@ type GatewaySpec struct {
 	//
 	// Support: Core
 	//
-	// +required
 	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=16
```

Comment:
MaxItems: 128

Reply:
Rob, I thought through this a bit more and I think start with a lower limit of 32 or 64.
If you really think about this, 64 listeners are a lot from a controller's perspective. I really doubt even large use cases will run into this limit because if you have a lot of listeners, operators like to split them out into multiple Gateways to avoid a very large blast radius.

Reply:
That's a good point. I've lowered this to 64.